static/js/pages/applications/ApplicationDashboardPage_test.js

```diff
@@ -385,6 +387,50 @@ describe("ApplicationDashboardPage", () => {
         })
       })
     })
+    ;[
+      [false, "normal user"],
+      [true, "alumni user"]
+    ].forEach(([isAlumni, desc]) => {
+      it(`show application steps for the ${desc}`, async () => {
+        const newApplication = Object.assign({}, application)
+        newApplication.bootcamp_run.allows_skipped_steps = isAlumni ?
+          true :
+          false
+
+        let newApplicationDetail = Object.assign({}, applicationDetail)
+        newApplicationDetail = setToAwaitingPayment(newApplicationDetail)
+        newApplicationDetail.user.profile.can_skip_application_steps = isAlumni ?
+          true :
+          false
+        const props = {
+          ...defaultProps,
+          applications:         [newApplication],
+          allApplicationDetail: {
+            [application.id]: newApplicationDetail
+          },
+          currentUser: isAlumni ? makeCompleteAlumniUser() : makeCompleteUser()
+        }
+
+        const wrapper = await renderExpanded(props)
+        const profileDetail = wrapper.find("ProfileDetail")
+        const resumeDetail = wrapper.find("ResumeDetail")
+        const videoInterviewDetail = wrapper.find("VideoInterviewDetail")
+        const reviewDetail = wrapper.find("ReviewDetail")
+        if (isAlumni) {
+          assert.isFalse(profileDetail.exists())
+          assert.isFalse(resumeDetail.exists())
+          assert.isFalse(videoInterviewDetail.exists())
+          assert.isFalse(reviewDetail.exists())
+        } else {
+          assert.isTrue(profileDetail.exists())
+          assert.isTrue(resumeDetail.exists())
+          assert.isTrue(videoInterviewDetail.exists())
+          assert.isTrue(reviewDetail.exists())
+        }
```

Comment:
This can be edited down

```suggestion
        assert.equal(profileDetail.exists(), isAlumni)
        assert.equal(resumeDetail.exists(), isAlumni)
        assert.equal(videoInterviewDetail.exists(), isAlumni)
        assert.equal(reviewDetail.exists(), isAlumni)
```